sequencing_report_service/services/local_runner_service.py

```diff
@@ -52,57 +47,72 @@ def __init__(self, job_repo_factory, nextflow_command_generator, nextflow_log_di
         self._job_repo_factory = job_repo_factory
         self._nextflow_jobs_factory = nextflow_command_generator
         self._nextflow_log_dirs = nextflow_log_dirs
-        self._currently_running_job = None
-        self._current_nxf_log = None
-        self._current_nxf_log_fh = None
 
-    def _start_process(self, job):
+    async def _start_process(self, job_id):
         with self._job_repo_factory() as job_repo:
-            log.debug("Will start command: %s", job.command)
+            job = job_repo.get_job(job_id)
             sys_env = os.environ.copy() or {}
             job_env = job.environment or {}
             env = {**sys_env, **job_env}
 
-            working_dir = os.path.join(self._nextflow_log_dirs, str(job.job_id))
+            working_dir = os.path.join(
+                self._nextflow_log_dirs, str(job.job_id))
```

Comment:
Not very important, but I guess job_id could be used here? 
```suggestion
                self._nextflow_log_dirs, job_id)
```